pallets/aleph/src/lib.rs

```diff
@@ -46,7 +47,27 @@ pub mod pallet {
     #[pallet::hooks]
     impl<T: Config> Hooks<BlockNumberFor<T>> for Pallet<T> {
         fn on_runtime_upgrade() -> frame_support::weights::Weight {
-            migrations::v0_to_v1::migrate::<T, Self>() + migrations::v1_to_v2::migrate::<T, Self>()
+            let on_chain = <Pallet<T> as GetStorageVersion>::on_chain_storage_version();
+            let current = 2;
+            let prev = 1;
+            let genesis = 0;
+            T::DbWeight::get().reads(1)
+                + match on_chain {
+                    _ if on_chain == current => 0,
```

Comment:
perhaps just use explicit versions, w-out storing them in a variable?

Reply:
Changed.